lib/srv/uacc/uacc.go

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
this still needs OS-specific handling - if it's common for darwin/linux, it won't compile on windows and it's supported on windows at least for tsh iiuc.

Reply:
I've just added a shim module that will disable uacc on operating systems where we don't support it so it'll still compile.

Reply:
Does that seem good?